test/bond/test/james.cljc

```
@@ -11,11 +12,13 @@
   (bond/with-spy [foo]
     (foo 1)
     (foo 2)
-    (let [exception (is (thrown? clojure.lang.ArityException (foo 3 4)))]
-      (is (= [{:args [1] :return 2}
-              {:args [2] :return 4}
-              {:args [3 4] :throw exception}]
-             (bond/calls foo))))))
+    (is (= [{:args [1] :return 2}
+            {:args [2] :return 4}]
+           (bond/calls foo)))
+    ;; cljs doesn't throw artity exceptions
```

~ What is the cljs behaviour? Does it silently accept the wrong arity?